### ISSUE TITLE
Improve documentation of workflow_run

### ISSUE THREAD

**Issue by makeable** [github/docs#23257]
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

https://docs.github.com/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#workflow_run
workflow_run is not well documented. There is NO information on what is contained in the workflow_run object in the payload

### What part(s) of the article would you like to see updated?

I would like to see documentation explaining what is contained in the object. At present it just says "required"

### Additional information

This has been raised a number of times and just gets closed.

**Comment by cmwilson21** [comment]
@makeable Thanks so much for opening an issue! I'll triage this for the team to take a look :eyes:

**Comment by skedwards88** [comment]
Thanks for opening this issue! This looks like a bug that we need to fix internally, so I'm going close this issue and open an internal issue for our engineering team.

**Comment by rebwill** [comment]
I am not sure what fix was made on this a few months back, but the `workflow_run` event payload documentation is still lacking any reference for the `workflow_run` object.

About two weeks ago, I briefly saw a dropdown [here](https://docs.github.com/en/webhooks-and-events/webhooks/webhook-events-and-payloads#workflow_run) under the `workflow_run` entry -- similar to what appears under the properties of `workflow_job.deployment`. However that dropdown disappeared the next time I reloaded the page, and I have not seen it again. I tried in Firefox and Chrome in case it was a browser issue, but no luck.

I believe this issue should be reopened.
